packages/gatsby/src/query/queue.ts

```diff
@@ -62,7 +62,7 @@ const createDevelopQueue = (getRunner: () => GraphQLRunner): Queue => {
           if (!queryJob.isPage) {
             websocketManager.emitStaticQueryData({
               result,
-              id: queryJob.id,
+              id: queryJob.hash,
```

Comment:
We actually use `hash` when storing static query results. Previously it was working because `id` was equal to `hash` but now it's not the case.